string Class
float64 probability
geometry_msgs/Point position
geometry_msgs/Point position_m
